packages/web-frontend/src/containers/EnlargedDialog/EnlargedDialog.js

```diff
@@ -133,6 +173,11 @@ const EnlargedDialogComponent = ({
         scroll={isMobile() ? 'body' : 'paper'}
         PaperProps={{ style: getDialogStyle() }}
       >
+        {isLoading && (
```

Comment:
I believe that this sort of thing would look better than what we used to have, which was just a grey filter over the whole screen.

Reply:
Even better would be just a loading symbol in place of the report itself, and still show the date range picker, close/back icons and title (as we should have that information).

Reply:
We will need to change the loader somehow so that it only shows when the dropdown level changes and not when the date changes. There is already a loader for the date changing next to the date picker and then the chart themselves don't have a loader so that the chart animation can happen. We are keen to keep the chart animation.

Reply:
Also, in the existing drill down the container overflow is set to scroll and now it changes sizes. We need to fix this so that the dialog doesn't keep changing size. It might be a matter of just adding a max-height somewhere?

Reply:
> We will need to change the loader somehow so that it only shows when the dropdown level changes and not when the date changes.
👍 

> We need to fix this so that the dialog doesn't keep changing size.

In investigating this, it looks like the only reason it used to be the same size is that the underlying chart was still being rendered and setting the size, and the `drillDownOverlay` just conformed to that. 

My understanding has been completely changed. Instead of navigating to a new, related report, we were showing some deeper information over the top of the chart. 

Perhaps the best way to go about this is to revert specifically the drillDownOverlay changes? Now that I understand the old system better, it may not have needed a refactor at all? We can still delete all the redux stuff? What do you think?

Reply:
hmmm if you are happy to do that then that sounds like a good idea to me. and yep if we can still remove the redux stuff that would be good.

Reply:
This little bit of gap at the sides and transparency matches my new understanding... 
It definitely needs a back button, but after that do you think it's more clear?
![image](https://user-images.githubusercontent.com/32168886/100307929-2d67b480-2ffb-11eb-97a8-5f70bee7faf7.png)

Reply:
I'm going to commit here but make it match the existing functionality, just thought this might be worth a look.